src/order_service/models/check_order_service_db.py

```diff
@@ -0,0 +1,37 @@
+from sqlalchemy import inspect
+from sqlalchemy.orm import Session
+
+from src.order_service.models.init_data import init_statuses
+from .database import engine, Base, create_tables
+from .models import Status
+
+def database_connection():
+    try:
+        with engine.connect():
+            print('Усешно подключенно к БД')
+    except Exception as e:
+        print(f'Ощибка: {e}')
+        return
+
+def check_statuses():
+    with Session(engine) as session:
+        statuses = session.query(Status).all()
+        if not statuses:
+            print('Таблица пуста')
+        for status in statuses:
+            print(f'ID: {status.id}, Type: {status.type}')
+
+def check_db_structure():
```

Comment:
зачем нужна эта функция? проинспектировать работу можно просто через sqlite

Reply:
предлагаю уходить от принтов без повода. 
давай в случае ошибки выбрасывать исключение, в случае если что-то не так его выбрасывать

```suggestion

class DatabaseInitializationError(Exception):
    """Database not inited"""
    pass

raise DatabaseInitializationError from e
```

если что-то не так, наша программа должна выключаться и не продолжать работу. полагаться на принты недостаточно в дальнейшем. 

прочитай про подход FailFast и FailSafe обязательно. пора познакомиться с ним